Detectors/TRD/simulation/src/Digitizer.cxx

```diff
@@ -120,10 +120,10 @@ void Digitizer::flush(DigitContainer& digits, o2::dataformats::MCTruthContainer<
   for (int det = 0; det < kNdet; ++det) {
     std::move(mDigitsCollection[det].begin(), mDigitsCollection[det].end(), std::back_inserter(digits));
     for (const auto& it : mSignalsMapCollection[det]) {
-      std::move(it.second.labels.begin(), it.second.labels.end(), std::back_inserter(mMergedLabels));
+      labels.addElements(labels.getIndexedSize(), it.second.labels); 
     }
   }
-  labels.addElements(labels.getIndexedSize(), mMergedLabels);
+  //moved above to almost fix labels labels.addElements(labels.getIndexedSize(), mMergedLabels);
```

Comment:
Please, undo this changes, this doesn't fix the labels at all. I am fixing this in another PR.

Reply:
it fixes it in so far as i dont get *all* the labels on the first digit, which messes up the trap simulator. this was discussed last thursday.

Reply:
hang on this was removed from this pr? let me check, to go in later.

Reply:
out, it was reverted already in a previous push, not sure how it crept back in.